starter-guides/browser/README.md

```diff
@@ -0,0 +1,26 @@
+# Getting Started with a Lit SDK Browser Implementation
+
+This is a very simple example repository for getting started with a browser implementation of the Lit SDK. All it will do is connect an instance of the `LitNodeClient` to the Lit Network. If you are searching for more developed uses of Lit, the [Developer Guides Code](https://github.com/LIT-Protocol/developer-guides-code) repository is for you. 
```

Comment:
```suggestion
This is a very simple example repository for getting started with a browser implementation of the Lit SDK. All it will do is connect an instance of the `LitNodeClient` to the Lit Network. If you are searching for more developed uses of Lit, check out the other guides in this repository. 
```

Reply:
Completely missed this, my bad